src/ItemTable.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { useGetItemsQuery } from '../app/services/api';
+import { Button } from 'primereact/button';
+import { Dialog } from 'primereact/dialog';
+import { InputText } from 'primereact/inputtext';
+import { MultiSelect } from 'primereact/multiselect';
+import { DataTable, DataTableSortEvent } from 'primereact/datatable';
+import { Column } from 'primereact/column';
+
+interface Item {
+  id: number;
+  n: string;
+  c: string[];
+}
+
+const ItemTable: React.FC = () => {
+  const { data: items = [] } = useGetItemsQuery(undefined); 

```

Comment:
Why do you pass `undefined` if the function does not expect any arguments? (`void` in the interface)